### PR TITLE
clariy multihop depth

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr clarifies multi-hop depth and whether more than 3 joins are supported in mf. 

stemmed from users asking and this slack thread: https://dbt-labs.slack.com/archives/C03KHQRQUBX/p1708571327378159?thread_ts=1708570486.187929&cid=C03KHQRQUBX

### CODE REVIEW

File: website/docs/docs/build/join-logic.md [hunk 1]
```diff
@@ -84,17 +84,15 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
 
 ## Multi-hop joins
 
-:::info
-MetricFlow can join three tables at most, supporting multi-hop joins with a limit of two hops.
-:::
+MetricFlow allows users to join measures and dimensions across a graph of entities, referred to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
 
-MetricFlow allows users to join measures and dimensions across a graph of entities, which we refer to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
+MetricFlow can join up to three tables, supporting multi-hop joins with a limit of two hops. This enables complex data analysis without ambiguous paths. It supports navigating through data models, like moving from `orders` to `customers` to `country` tables. While direct three-hop paths are limited to prevent confusion from multiple routes to the same data, MetricFlow does allow joining more than three tables if the joins don’t exceed two hops to reach a dimension.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]: hey @Jstein77 - does this sound right?

File: website/docs/docs/build/join-logic.md [hunk 2]
```diff
@@ -84,17 +84,18 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
...
 
-MetricFlow allows users to join measures and dimensions across a graph of entities, which we refer to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
+MetricFlow can join up to three tables, supporting multi-hop joins with a limit of two hops. This does the following:
+- Enables complex data analysis without ambiguous paths.
+- Supports navigating through data models, like moving from `orders` to `customers` to `country` tables.
+- While direct three-hop paths are limited to prevent confusion from multiple routes to the same data, MetricFlow does allow joining more than three tables if the joins don’t exceed two hops to reach a dimension. 
 
-Here's an example schema for reference:
+For example, if you have two models, `country` and `region`, where customers are linked to countries, which in turn are linked to regions. You can join all of them in a single SQL query and can dissect `orders` by `customer__country_country_name` but not by `customer__country__region_name`.
```
Comment (matthewshaver) [inline website/docs/docs/build/join-logic.md]: The first part is sort of an incomplete thought if you write it as two sentences
```suggestion
For example, if you have two models, `country` and `region`, where customers are linked to countries, which in turn are linked to regions, you can join all of them in a single SQL query and can dissect `orders` by `customer__country_country_name` but not by `customer__country__region_name`.
```

File: website/docs/docs/build/join-logic.md [hunk 3]
```diff
@@ -84,17 +84,18 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
 
 ## Multi-hop joins
 
-:::info
-MetricFlow can join three tables at most, supporting multi-hop joins with a limit of two hops.
-:::
+MetricFlow allows users to join measures and dimensions across a graph of entities, referred to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
```
Comment (matthewshaver) [inline website/docs/docs/build/join-logic.md]:
```suggestion
MetricFlow allows users to join measures and dimensions across a graph of entities, referred to as a 'multi-hop join' because users can move from one table to another like a 'hop' within a graph.
```

File: website/docs/docs/build/join-logic.md [hunk 4]
```diff
@@ -84,17 +84,18 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
...
 
-Here's an example schema for reference:
+For example, if you have two models, `country` and `region`, where customers are linked to countries, which in turn are linked to regions. You can join all of them in a single SQL query and can dissect `orders` by `customer__country_country_name` but not by `customer__country__region_name`.
 
-![Multi-Hop-Join](/img/docs/building-a-dbt-project/multihop-diagram.png)
+![Multi-Hop-Join](/img/docs/building-a-dbt-project/multihop-diagram.png "Example schema for reference")
 
-Notice how this schema can be translated into the three MetricFlow semantic models below to create the metric 'Average purchase price by country' using the `purchase_price` measure from the sales table and the `country_name` dimension from the `country_dim` table.
+Notice how the schema can be translated into the following three MetricFlow semantic models to create the metric 'Average purchase price by country' using the `purchase_price` measure from the sales table and the `country_name` dimension from the `country_dim` table.
```
Comment (matthewshaver) [inline website/docs/docs/build/join-logic.md:99]: Maybe I'm reading the next section incorrectly, but there only appear to be two semantic models in the example. Is there a `-name:` field missing?

File: website/docs/docs/build/join-logic.md [hunk 5]
```diff
@@ -84,17 +84,18 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
...
-:::info
-MetricFlow can join three tables at most, supporting multi-hop joins with a limit of two hops.
-:::
+MetricFlow allows users to join measures and dimensions across a graph of entities, referred to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
 
-MetricFlow allows users to join measures and dimensions across a graph of entities, which we refer to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
+MetricFlow can join up to three tables, supporting multi-hop joins with a limit of two hops. This does the following:
+- Enables complex data analysis without ambiguous paths.
+- Supports navigating through data models, like moving from `orders` to `customers` to `country` tables.
+- While direct three-hop paths are limited to prevent confusion from multiple routes to the same data, MetricFlow does allow joining more than three tables if the joins don’t exceed two hops to reach a dimension.
```
Comment (matthewshaver) [inline website/docs/docs/build/join-logic.md]: This last point feels a little out of place. Should it be a follow-up note or maybe reworded?
Reading aloud:
This does the following... enables ✅... supports✅... while direct...

File: website/docs/docs/build/join-logic.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/build/join-logic.md:99]: oh i think this is missing the 3rd country one:

  - name: country # This is the third semantic model
    entities:
      - name: country_id
        type: primary
    dimensions:
      - name: country_name
        type: categorical
        
        
 I'll add it in!

Comment (mirnawong1) [reply website/docs/docs/build/join-logic.md:99]: good catch!

File: website/docs/docs/build/join-logic.md [hunk 6]
```diff
@@ -84,17 +84,19 @@ mf query --metrics average_purchase_price --dimensions metric_time,user_id__type
 
 ## Multi-hop joins
 
-:::info
-MetricFlow can join three tables at most, supporting multi-hop joins with a limit of two hops.
-:::
+MetricFlow allows users to join measures and dimensions across a graph of entities, referred to as a 'multi-hop join' because users can move from one table to another like a 'hop' within a graph.
```
Comment (mirnawong1) [inline website/docs/docs/build/join-logic.md]:
```suggestion
MetricFlow allows users to join measures and dimensions across a graph of entities by moving from one table to another within a graph. This is referred to as "multi-hop join". 
```

**Review by matthewshaver — APPROVED** [review]
